### PR TITLE
don't save plugins assets

### PR DESCRIPTION
**Pull request by saeideng**
a solution for #578

### CODE REVIEW

**Comment by saeideng** [comment]
another option could be add a button to hide this and +  pagination requests

File: src/ToolbarService.php [hunk 1]
```diff
@@ -214,6 +215,15 @@ public function saveData(ServerRequest $request, ResponseInterface $response)
         ) {
             return null;
         }
+        // Skip plugins assets
+        if (strpos($path, '.') !== false &&
+            strpos($path, '/.') === false &&
+            strpos($path, '..') === false &&
+            $response->getStatusCode() === 200 &&
+            $this->checkFileExists($path)
```
Comment (markstory) [inline src/ToolbarService.php]: Instead looking for the file, could we skip all css, js and common image files? The checks would be simpler.

Comment (saeideng) [reply src/ToolbarService.php]: first I wanted to not catch all files that not served by AssetMiddleware
https://github.com/cakephp/cakephp/blob/61a1ff9672430456a7a261f053911cab93793742/src/Routing/Middleware/AssetMiddleware.php#L86-L93
and then people should see other requests that are 404 not exists
just like the main assets behaviors in DebugKit 

but if `$response->getStatusCode() === 200 &&` is enough probably we could do this method simpler

**Comment by dereuromark** [comment]
Do we want to proceed here? Incremental Improvements?

**Comment by saeideng** [comment]
closing in favor of #764

**Pull request closed without merging**